Index: Augmentation/LineAugmentation.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import numpy as np\r\nimport imgaug as ia\r\nimport imgaug.augmenters as iaa\r\nimport cv2\r\nfrom wezel.main import assert_dir\r\n\r\nia.seed(1)\r\n\r\n\r\ndef read_images(path):\r\n    srcimage = cv2.imread(path, cv2.IMREAD_GRAYSCALE)\r\n\r\n\r\nseq = iaa.Sequential([\r\n\r\n])\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Augmentation/LineAugmentation.py b/Augmentation/LineAugmentation.py
--- a/Augmentation/LineAugmentation.py	(revision f5dffbe674c8d913ae8ba5ca2fcd0682289a0990)
+++ b/Augmentation/LineAugmentation.py	(date 1684493848440)
@@ -8,8 +8,8 @@
 
 
 def read_images(path):
-    srcimage = cv2.imread(path, cv2.IMREAD_GRAYSCALE)
-
+    #srcimage = cv2.imread(path, cv2.IMREAD_GRAYSCALE)
+    cv2.imread(path,-1)
 
 seq = iaa.Sequential([
 
Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
new file mode 100644
--- /dev/null	(date 1684493411710)
+++ b/.gitignore	(date 1684493411710)
@@ -0,0 +1,1 @@
+/wezel/.idea/
